modulith/commodityFinder/build.gradle

```diff
@@ -1,24 +1,44 @@
 plugins {
+    id 'java'
+    id 'idea'
+    id "org.gradlex.extra-java-module-info" version "1.4"
 }
-
 dependencies {
     implementation project(':util')
     implementation project(':mybatis-util')
     implementation project(':messageprocessor-lib')
-    implementation 'org.springframework.boot:spring-boot-starter'
+    implementation 'org.springframework.boot:spring-boot-starter-jdbc:3.1.0'
```

Comment:
not a huge fan of all the versions in the dependencies block.

maybe move these an `ext {}` block so the are bundled ?

Reply:
moved do constraints block